cmd/otelcol/config/collector/config.d.linux/extensions/k8s-observer.discovery.yaml

```diff
@@ -0,0 +1,6 @@
+#####################################################################################
+# This file is generated by the Splunk Distribution of the OpenTelemetry Collector. #
+#####################################################################################
+# k8s_observer:
+#   enabled: true
+#   auth_type: serviceAccount
```

Comment:
Just realized that observer properties aren't nested in `config` mappings, which I'll update in a follow-up PR for consistency.